### ISSUE TITLE
Webhooks in endpoints.yml

### ISSUE THREAD

**Issue by shotamakino** [RasaHQ/rasa#4005]
**Description of Problem**:
Currently rabbitmq and kafka are supported for consuming events. These are configurable through the endpoints.yml. For simple integrations such as sending events off to an endpoint directly, setting up rabbitmq/kafka and consuming those events can be overkill.

**Overview of the Solution**:
I am hoping to add a new configuration to endpoints.yml under a property 'webhooks' where we can specify URLs that rasa posts events to.

**Definition of Done**:
- [ ] Tests are added
- [ ] Feature is documented

**Comment by akelad** [comment]
@shotamakino could you clarify what you mean? what kind of webhooks would you like?

**Comment by shotamakino** [comment]
So exploring some more, I've been able to accomplish what I required through a custom EventChannel by publishing a package and specifying this class through the endpoints.yml. 

What I was originally seeking for was a new EventChannel type with a url arg that specifies where to post the event to as a body parameter.

An alternative could be to allow the user to create a new module in the directory with their rasa configurations and specify this module via the endpoints.yml through the type property.
